torch/_inductor/codegen/cpp_template_kernel.py

```diff
@@ -98,7 +98,9 @@ def def_kernel(
         )
         sizevars = sorted(unique_sizevars, key=str)
         for sizevar in sizevars:
-            self.args.sizevars[sizevar] = f"k{sizevar}"
+            # For BMM, ensure that sizevars from previous codegen are not overwritten
+            if sizevar not in self.args.sizevars:
+                self.args.sizevars[sizevar] = f"k{sizevar}"
```

Comment:
Although we `define kernel` 3 times in BMM Template, I am not sure in which case we may overwritten the size with a different symbol. Feels like all the sizevar should be corresponding to a same symbol inside kernel.